src/engines/html5/html5.js

```diff
@@ -369,6 +369,12 @@ export default class Html5 extends FakeEventTarget implements IEngine {
     this._removeCueChangeListener();
   }
 
+  disableAllTextTracks(): void{
```

Comment:
we need it because the im letting the external controller handle the text track change  when it we use the external caption. so it needs to disable all the other text tracks. already added to HLS, will do it in dash as well
f